ocaml/testsuite/tests/typing-local/local.ml

```diff
@@ -393,25 +393,25 @@ module type F =
 
 let app1 (f : a:int -> b:local_ int ref -> unit -> unit) = f ~b:(local_ ref 42) ()
 [%%expect{|
-Line 1, characters 64-79:
+Line 1, characters 72-78:
 1 | let app1 (f : a:int -> b:local_ int ref -> unit -> unit) = f ~b:(local_ ref 42) ()
-                                                                    ^^^^^^^^^^^^^^^
+                                                                            ^^^^^^
```

Comment:
The original error message seems more correct. How I understand the situation: we have `ref 42`, which is later weakened to `local`, and the resulted value cannot escape its region. Therefore, the error message should be about `local_ ref 42`.

(Same for other similar changes).